src/rpc/misc.cpp

```diff
@@ -119,7 +119,6 @@ class DescribeAddressVisitor : public boost::static_visitor<UniValue>
         obj.pushKV("isscript", false);
         if (pwalletMain && pwalletMain->GetPubKey(keyID, vchPubKey)) {
             obj.pushKV("pubkey", HexStr(vchPubKey));
-            obj.pushKV("pubkeyhash", keyID.ToString());
```

Comment:
This should be retained, it was added in commit d26b937b76d74ee55ef6c83469c83ae8a625ab1d

Reply:
done